.travis.yml

```diff
@@ -31,7 +31,8 @@ cache:
 
 before_install:
   - sudo apt-get update -qq
-  - sudo apt-get install make gcc g++ python3-ply ncurses-dev uglifyjs libc6-dev-i386 python3-yaml -qq
+  - sudo apt-get install make gcc g++ python3-ply ncurses-dev uglifyjs libc6-dev-i386 python3-pip python3-yaml -qq
+  - sudo pip3 install pyelftools
```

Comment:
I had to use sudo -H pip3 install here, otherwise i get a error msg

Reply:
It appears to work fine in Travis, so I don't know that it needs to change; probably a configuration issue.